src/app/pages/home/containers/home-page.component.html

```diff
@@ -24,6 +24,22 @@
     >
     </task-progress>
   </div>
+  <div class="progress-report-container" *ngIf="progressReportEnabled">
+    <ion-fab
+      left
+      bottom
+      class="progress-report-button"
+      [class.clinical-button-bottom]="showCalendar"
+    >
+      <button ion-fab class="fab" (click)="showProgressReportPage()">
+        <svg xmlns='http://www.w3.org/2000/svg' class='ionicon' viewBox='0 0 512 512'>
```

Comment:
For readibility, would it be possible to export the svg to a separate file like how we do it with `assets/imgs/radar-logo.svg`?

Reply:
Yes, for sure.